critical/templates/critical/critical.html

```diff
@@ -0,0 +1,29 @@
+{# See: http://brianflove.com/2016/07/21/preloading-with-critical-path-css/ #}
+{# Author: Norman Rusch #}
+
+{% if critical_css %}
+	{% block critical_css %}
+		<style type="text/css">{% block critical_css_inline %}{{ critical_css|safe }}{% endblock %}</style>
+	{% endblock %}
+	{% block preload %}
+		<link rel="preload" as="style" type="text/css" href="{{ css_path }}" onload="this.onload=null;this.rel='stylesheet'" />
+	{% endblock %}
+	{% block noscript %}
+		<noscript><link rel="stylesheet" type="text/css" href="{{ css_path }}?noscript" /></noscript>
```

Comment:
I think we can drop that `noscript`-get param. I can not remember why I've added this here.

```suggestion
		<noscript><link rel="stylesheet" type="text/css" href="{{ css_path }}" /></noscript>
```